### PR TITLE
Node: Add randomUUID function to crypto module

### PR DESCRIPTION
**Pull request by bondz**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://nodejs.org/dist/latest-v14.x/docs/api/crypto.html#crypto_crypto_randomuuid_options>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by SimonSchick** [comment]
I currently have a PR out that adds this: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/52732 I'm currently waiting for maintainers to approve the PR chain that brings typings back up to date.

I'd suggest adding this in the node v14 typings rather than in v15 (current) as in v15 this change was introduced 15.6 which is far ahead of v15.0 (current typings).

**Comment by bondz** [comment]
Thanks @SimonSchick

updated.

**Review by galkin — APPROVED** [review]
LFGM

File: types/node/test/crypto.ts
Comment (SimonSchick) [inline types/node/test/crypto.ts:359]: This should be in the v14 folder as well.

Comment (bondz) [reply types/node/test/crypto.ts]: Missed that. Fixed.

Thanks.

**Review by peterblazejewicz — COMMENTED** [review]
For v14, should the header be updated? Otherwise it will fail at runtime,
e.g in LTS version I'm using:
```ts
// Type definitions for non-npm package Node.js 14.14
```
It should read 14.17, shoudn't it?

**Comment by bondz** [comment]
@peterblazejewicz I updated it.

**Review by SimonSchick — APPROVED** [review]

**Review by peterblazejewicz — APPROVED** [review]

**Review by galkin — APPROVED** [review]

**Review by RDIL — APPROVED** [review]

**Pull request merged**
